src/drivers/imx/edma.c

```diff
@@ -338,8 +338,8 @@ static int edma_setup_tcd(struct dma_chan_data *channel, int16_t soff,
 	 */
 	if (!dma_chan_reg_read(channel, EDMA_CH_MUX))
 		(channel->direction == DMA_DIR_MEM_TO_DEV) ?
-			dma_chan_reg_write(channel, EDMA_CH_MUX, IMX8ULP_DMAMUX2_SAI5_TX) :
-			dma_chan_reg_write(channel, EDMA_CH_MUX, IMX8ULP_DMAMUX2_SAI5_RX);
+			dma_chan_reg_write(channel, EDMA_CH_MUX, imx8ulp_dmamux_tx_int) :
+			dma_chan_reg_write(channel, EDMA_CH_MUX, imx8ulp_dmamux_rx_int);
```

Comment:
These variables seem to be uninitialised? As external variables they'll be implicitly initialised to 0, is that what's needed here? In other words, I think this commit should be merged with the next one.

Reply:
@TangleZ I think we should hide some information about dmamux_tx_int/rx_int inside dma_chan_data (maybe priv_data or dev_data).

I wonder if something like this will work:

```
dma_chan_reg_write(channel, EDMA_CH_MUX, channel->dev_data->dai->dai_index == 5 ? IMX8ULP_DMAMUX2_SAI5_TX : IMX8ULP_DMAMUX2_SAI6_TX)
```